packages/server/computation_container/random/random.hpp

```diff
@@ -45,3 +46,53 @@ class RandGenerator
         return ret;
     }
 };
+
+template <typename CSPRNG>
+class random_csprng
+{
+    CSPRNG prng;
+
+public:
+    random_csprng() : prng() {}
+    /**
+     * @brief 上限と下限を指定する
+     *
+     * @param min 下限
+     * @param max 上限
+     * @return 範囲内の乱数生成
+     */
+    template <typename Result>
+    auto get(
+        Result min = std::numeric_limits<Result>::min(),
+        Result max = std::numeric_limits<Result>::max()
+    )
+    {
+        // integral
+        if constexpr (std::is_integral_v<Result>)
+        {
+            std::uniform_int_distribution<Result> dist(min, max);
+            return dist(prng);
+        }
+        // float
+        else
+        {
+            std::uniform_real_distribution<Result> dist(min, max);
+            return dist(prng);
+        }
+    }
+
+    template <typename Result>
+    auto make_array(
```

Comment:
It would be my preference, but I would like to match the name with a scalar function. What do you think?
`get`,`get_array` or `make`, `make_array`

Reply:
indeed.
i ll rename get~.